casepro/msgs/models.py

```diff
@@ -132,6 +133,125 @@ def __str__(self):
 
 
 @python_2_unicode_compatible
+class Language(models.Model):
```

Comment:
What's the reason for modelling languages in the database rather than just storing codes and getting the actual language name from pycountry/iso639? 

We currently have a `Contact.language` field which is an ISO639-2 code and uses [iso639](https://pypi.python.org/pypi/iso639) to get the name for display. We can tweak that so that different encodings are supported (we're only using (ISO639-2 because that's what RapidPro uses), but I think we want `FAQ.language` and `Contact.language` to be modelled in the same way.